pkg/scalers/rabbitmq_scaler.go

```diff
@@ -494,3 +495,10 @@ func getMaximum(q []queueInfo) (int, float64) {
 	}
 	return maxMessages, maxRate
 }
+
+// Mask host for log porpouses
+func (s *rabbitMQScaler) anonimizeRabbitMQError(err error) error {
+	errorMessage := fmt.Sprintf("error inspecting rabbitMQ: %s", err)
+	m1 := regexp.MustCompile(`([^ \/:]+):([^\/:]+)\@`)
+	return fmt.Errorf(m1.ReplaceAllString(errorMessage, "user:password@"))
```

Comment:
I'm not experienced with Go, but does this obfuscate the user name as well? If so, why? I don't see a problem with outputting that which can help troubleshooting.

If not, no problem.

Reply:
I did it intentionally because we let the domain and the path (with the regex and/or the virtual host and queue) for troubleshooting, but I can update the regex to replace only the password
wdyt?

Reply:
What do you think @zroubalik ?

Reply:
I fine with both, I don't mind the `username` being obfuscated.

Reply:
so? You have the last word @tomkerkhove xD

Reply:
what do you think if I add a test case to ensure that the rest of the host is not obfuscated?
Maybe with the domain + path + virtual host is enough to discover which trigger is failing :thinking:

Reply:
Ship it 🐿

Reply:
I will resolve this conversation in that case :)